vllm/v1/worker/tpu_model_runner.py

```diff
@@ -139,10 +138,8 @@ def __init__(
                                             device="cpu")
         self.slot_mapping_np = self.slot_mapping_cpu.numpy()
 
-        padded_max_num_blocks_per_req = _get_padded_number(
-            self.max_num_blocks_per_req, NUM_KV_PAGES_PER_BLOCK)
         self.block_table_cpu = torch.zeros(
-            (self.max_num_tokens, padded_max_num_blocks_per_req),
+            (self.max_num_tokens, self.max_num_blocks_per_req),
```

Comment:
What are the contraints of the kernel now? Multiple of 2?

Reply:
block_table.shape[1] doesn't have any constraint now.

Reply:
Do we not need any changes to the slicing logic into this buffer during execution?